rust-cargo.el

```diff
@@ -67,46 +67,52 @@
 
 ;;; Internal
 
-(defun rust--compile (format-string &rest args)
+(defun rust--compile (comint format-string &rest args)
   (when (null rust-buffer-project)
     (rust-update-buffer-project))
   (let ((default-directory
           (or (and rust-buffer-project
                    (file-name-directory rust-buffer-project))
               default-directory)))
-    (compile (apply #'format format-string args))))
+    (compile (apply #'format format-string args) (not (not comint)))))
```

Comment:
Isn't `(not (not comint))` is same as `comint` ?

Reply:
Not in ELisp :), this is needed to ensure that `comint` argument is a boolean value, so that it works with
universal argument provided by `C-u`. I've added a comment about that in the source.

Reply:
Ah, got it. Thanks!